README.rst

```diff
@@ -41,40 +41,41 @@ aiogram |beta badge|
     :target: https://app.codecov.io/gh/aiogram/aiogram
     :alt: Codecov
 
-**aiogram** modern and fully asynchronous framework for
-`Telegram Bot API <https://core.telegram.org/bots/api>`_ written in Python 3.8 with
+**aiogram** is a modern and fully asynchronous framework for
+`Telegram Bot API <https://core.telegram.org/bots/api>`_ written in Python 3.8 using
 `asyncio <https://docs.python.org/3/library/asyncio.html>`_ and
 `aiohttp <https://github.com/aio-libs/aiohttp>`_.
 
-It helps you to make your bots faster and simpler.
+Make your bots faster and more powerful!
 
 .. danger::
 
-    **Breaking News:**
```

Comment:
this section is a play on the words "breaking changes", so this line shouldn't be changed

Reply:
got it, returned the line to its original version